extensions/CheckUser/api/ApiQueryCheckUser.php

```diff
@@ -152,14 +152,14 @@ public function execute() {
 				}
 
 				$this->addFields( array( 
-					'cuc_user', 'cuc_user_text', 'cuc_timestamp', 'cuc_ip', 'cuc_agent' ) );
+					'cuc_user', 'cuc_timestamp', 'cuc_ip', 'cuc_agent' ) );
 
 				$res = $this->select( __METHOD__ );
 				$result = $this->getResult();
 
 				$users = array();
 				foreach ( $res as $row ) {
-					$user = User::getUsername( $row->cuc_user, $row->cuc_user_text ); // SUS-2944
+					$user = User::getUsername( $row->cuc_user, $row->cuc_ip ); // SUS-2944, SUS-3080
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app